api/routes/apds/get.endpoint.js

```diff
@@ -85,7 +85,8 @@ describe('APD endpoint', () => {
         const cookies = await login();
 
         const { response, body } = await request.get(url(4000), {
-          jar: cookies
+          jar: cookies,
+          json: true
```

Comment:
By making this request `json`, it means when the snapshot doesn't match, it can show you exactly where it doesn't match instead of showing you the whole freakin' thing as if there's any chance in hell you'll be able to tell what changed.

This should have always been this way.

Reply:
MY HERO